langchain4j-agentic/src/main/java/dev/langchain4j/agentic/internal/AgentCall.java

```diff
@@ -0,0 +1,15 @@
+package dev.langchain4j.agentic.internal;
+
+import java.util.Arrays;
+
+public record AgentCall(String agentName, Object[] input, Object output) {
+
+    @Override
+    public String toString() {
+        return "AgentInvocation{" +
```

Comment:
```suggestion
        return "AgentCall{" +
```

Or sohuld the class be named `AgentInvocation` instead?

Reply:
It is exposed from the `Cognisphere` public API

Reply:
I renamed it `AgentInvocation` but also removed the method from the `Cognisphere` interface. It wasn't supposed to leak into public API and it was there only for testing purposes.